### ISSUE TITLE
Setting Base URL for Couchbase UI Behind Reverse Proxy

### ISSUE THREAD

**Issue by cjconnor24** [couchbase/docker#218]
**Description:**
We are running Couchbase on Docker in our developer environments and would like to move the Couchbase UI behind our reverse proxy for better accessibility.

**Current Access:**
Direct access: http://server.devdomain.com:8091/

**Desired Access:**
Through reverse proxy: https://server.devdomain.com/db/

**Problem:**
After configuring the reverse proxy, the Couchbase login page gets returned correctly.

![image](https://github.com/user-attachments/assets/3113574c-1328-48cc-a36f-ace48e2164b6)

![image](https://github.com/user-attachments/assets/d6ecf365-e4e2-4937-80e1-972e353c239c)

However, some parts of the UI try to re-route to the root domain, causing issues with loading JavaScript files and other resources.

![image](https://github.com/user-attachments/assets/37c9fde8-d4df-4579-aea2-cb62515aaaf7)

**Question:**
How can we set a base URL for Couchbase, perhaps via an environment variable or configuration property, so that all links and resources resolve correctly to the `/db` base path?

**Comment by ceejatec** [comment]
Hi @cjconnor24 - this repository is specifically for hosting our Dockerfiles, and isn't generally monitored for support requests. So I'll close this ticket.

For what it's worth, I've reproduced your symptom. I found an [older ticket](https://issues.couchbase.com/browse/MB-22816) which suggests this may have worked in the past, but it was from quite some time ago. Evidently something has broken it in the meantime. I will ask around internally to see if this is something we should support, and if so, I'll create a new ticket and update you here. In the meantime, however, I'm afraid this isn't going to work except at the root of a domain. You might consider using something like Traefik in your Docker deployments to enable hostname-based traffic redirection; we use it for a number of similar setups here.
